Sources/PackagePlugin/Context.swift

```diff
@@ -39,6 +39,12 @@ public struct PluginContext {
     /// an error if the tool cannot be found. The lookup is case sensitive.
     public func tool(named name: String) throws -> Tool {
         if let path = self.toolNamesToPaths[name] {
+            // For PluginAccessibleTool.builtTool the triples value is always nil
```

Comment:
Maybe add some clarification here that this is intentional since if we are able to build the tool, it is by definition supporting the target platform. I think the uninitiated reader could think this is describing an issue currently.